packages/web-app-files/src/mixins/deleteResources.js

```diff
@@ -160,9 +163,27 @@ export default {
         this.toggleModalConfirmButton()
 
         // Load quota
-        if (this.user?.id) {
```

Comment:
User should be always there, that's why I omit this condition

Reply:
No, deleting resources is possible on public links with edit permissions.

Reply:
But I have a location check here ?

Reply:
lol, nevermind. Sorry, location check was not visible on the front page of the PR where I stumbled over this comment.